docker-compose.yml

```diff
@@ -536,6 +556,10 @@ volumes:
   # Not being external will still persist data across restarts.
   # It won't persist if someone does a docker compose down -v.
   sentry-vroom:
+  # This volume stores task data that is inflight
+  # It should persist across restarts. If this volume is
+  # deleted, up to ~2048 tasks will be lost.
```

Comment:
This begs a question. If the volume if deleted, then how should one recover 2048 lost tasks?

Reply:
You would have to reset offsets in Kafka.